static/app/views/nav/optInBanner.tsx

```diff
@@ -30,9 +31,12 @@ export function OptInBanner({collapsed, organization}: Props) {
 
   return (
     <TranslucentBackgroundPanel isDarkMode={isDarkMode}>
-      <Title>{t('✨ Try New Navigation')}</Title>
+      <Title>
+        {t('New Navigation')}
+        <FeatureBadge type="alpha" />
+      </Title>
       <Description>
-        {t("We've redesigned our sidebar to make it easier to navigate Sentry.")}
+        {t('Say hello to your improved sidebar—less clutter, clearer workflows.')}
```

Comment:
should there be spaces around the hyphen? this almost reads like "sidebarless"

Reply:
You know what I think you're right, adding spacing looks a bit better